mypy/checkexpr.py

```diff
@@ -2451,6 +2451,14 @@ def lookup_definer(typ: Instance, attr_name: str) -> Optional[str]:
             else:
                 return result
 
+        # We finish invoking above operators and no early return happens. Therefore,
+        # we check if either the LHS or the RHS is Instance and fallbacks to Any,
+        # if so, we also return Any
+        if ((isinstance(left_type, Instance) and left_type.type.fallback_to_any) or
+                (isinstance(right_type, Instance) and right_type.type.fallback_to_any)):
+            any_type = AnyType(TypeOfAny.special_form)
```

Comment:
I'm not sure if `special_form` is the right thing here but I'm not totally sure what else to do. `from_another_any` is correct but digging out the other any is almost certainly not worth bothering.

Reply:
I am also not sure about whether `special_form` is the right thing to go, but clearly directly using `from_another_any` with either `left_type` or `right_type` here is incorrect. To proper use `from_another_any`, we need a better way to transform the Instance to an Any